gradle-failure-reports/src/main/java/com/palantir/gradle/failurereports/FailureReportsRootPlugin.java

```diff
@@ -84,12 +81,14 @@ private void collectOtherTaskFailures(Project project, TaskProvider<FinalizerTas
                                     projectTasks)
                             .filter(FailureReportsRootPlugin::isAllowedTask)
                             .filter(FailureReporterResources::executedAndFailed)
-                            .map(ThrowableFailureReporter::maybeGetFailureReport)
-                            .flatMap(Optional::stream))));
+                            .map(ThrowableFailureReporter::getFailureReport))));
         });
     }
 
     private static boolean isAllowedTask(Task task) {
-        return !(task instanceof JavaCompile) && !(task instanceof Checkstyle) && !(task instanceof FinalizerTask);
+        return !(task instanceof JavaCompile)
+                && !(task instanceof Checkstyle)
+                && !(task instanceof FinalizerTask)
+                && !task.getName().equals(VERIFY_LOCKS_TASK);
```

Comment:
Do we need to ignore this?

Reply:
I think with https://github.com/palantir/gradle-consistent-versions/pull/1132 we can remove the special casing for verifyLocks above (I guess this could be another PR)